pkg/cloudprovider/nodecontroller/nodecontroller.go

```diff
@@ -136,6 +137,61 @@ func NewNodeController(
 	}
 }
 
+// Generates num pod CIDRs that could be assigned to nodes.
+func generateCIDRs(num int) util.StringSet {
+	res := util.NewStringSet()
+	for i := 0; i < num; i++ {
+		// TODO: Make the CIDRs configurable.
+		res.Insert(fmt.Sprintf("10.244.%v.0/24", i))
```

Comment:
This should also be extended to accommodate large clusters. KUBE_GCE_CLUSTER_CLASS_B implies it will only ever be a /16, but I think @satnam6502 has been trying to test larger clusters, and GKE uses a /14.